### PR TITLE
Async ensnames

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-example/test/services/BackupService.js [hunk 1]
```diff
@@ -6,11 +6,14 @@ describe('BackupService', async () => {
   let identityService;
 
   before(async () => {
-    identityService = [];
-    identityService.identity = [];
-    identityService.identity.name = 'kyle';
+    identityService = {
+      identity: {
+        name: 'kyle'
```
Comment (alexvandesande) [inline universal-login-example/test/services/BackupService.js]: Is our identity service named "Kyle"?

Comment (marekkirejczyk) [reply universal-login-example/test/services/BackupService.js]: This is example data for testing. 
The name picked in a way to immortalize one of the great contributors of ours :)

File: universal-login-example/src/services/EnsNameService.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    this.ensService = ensService;
+    this.historyService = historyService;
+    this.running = false;
+    this.isWorking = false; 
+    this.tick = tick;
+  }
+
+  subscribe() {
+    this.running = true;
+    this.loop();
```
Comment (marekkirejczyk) [inline universal-login-example/src/services/EnsNameService.js]: What about a callback?

File: universal-login-example/src/services/EnsNameService.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  }
+  
+  async loop() {
+    do {
+      if (!this.isWorking) {
+        this.isWorking = true;
```
Comment (marekkirejczyk) [inline universal-login-example/src/services/EnsNameService.js]: isWorking don't seem to be used anywhere

**Pull request merged**